whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/AsBlockSpec.groovy

```diff
@@ -439,7 +439,7 @@ class AsBlockSpec extends BaseQueryUpdateSpec {
     def "modify as-block, change mnt-by, old mnt-by pw supplied, new mnt-by pw not supplied"() {
 
         given:
-            syncUpdate(getTransient("RIPE-DBM-STARTUP-MNT") + "password:startup")
+            syncUpdate(getTransient("RIPE-DBM-STARTUP-MNT") + "password:startup\noverride:       denis,override1")
```

Comment:
Why do you need to use override for this update? Does it change the test behaviour?

Reply:
A validation rule was added to prevent RIPE maintainers from being created unless override authentication is used.